tests/maxsize_rule_list_benchmark_test.go

```diff
@@ -0,0 +1,79 @@
+package tests
+
+import (
+	"math/rand"
+	"strconv"
+	"testing"
+
+	cb "github.com/alibaba/sentinel-golang/core/circuitbreaker"
+)
+
+func Test_Size_1000_Circuit_Breaker_Rules_Update(t *testing.T) {
+	rs := make([]cb.Rule, 0, 1000)
+	intervals := []uint32{10000, 15000, 20000, 25000, 30000}
+	for i := 0; i < 1000; i++ {
+		retryTimeout := intervals[rand.Int()%5]
```

Comment:
`rand.Int()` will generate the same pseudo-random sequence if you don't call `rand.Seed()` first.

Reply:
Updated.